executor/importer/import.go

```diff
@@ -392,75 +400,29 @@ func NewLoadDataController(plan *Plan, tbl table.Table, astArgs *ASTArgs) (*Load
 	return c, nil
 }
 
-func (e *LoadDataController) initFieldParams(plan *Plan) error {
+func (e *LoadDataController) checkFieldParams() error {
 	if e.Path == "" {
 		return exeerrors.ErrLoadDataEmptyPath
 	}
-	if e.Format != LoadDataFormatDelimitedData && e.Format != LoadDataFormatParquet && e.Format != LoadDataFormatSQLDump {
-		return exeerrors.ErrLoadDataUnsupportedFormat.GenWithStackByArgs(e.Format)
-	}
-
-	if e.FileLocRef == ast.FileLocClient {
-		if e.Detached {
-			return exeerrors.ErrLoadDataLocalUnsupportedOption.FastGenByArgs("DETACHED")
+	if e.InImportInto {
+		if e.Format != DataFormatCSV && e.Format != DataFormatParquet && e.Format != DataFormatSQL {
+			return exeerrors.ErrLoadDataUnsupportedFormat.GenWithStackByArgs(e.Format)
 		}
-		if e.Format == LoadDataFormatParquet {
-			// parquet parser need seek around, it's not supported for client local file
-			return exeerrors.ErrLoadParquetFromLocal
+	} else {
+		if e.NullValueOptEnclosed && len(e.FieldsEnclosedBy) == 0 {
+			return exeerrors.ErrLoadDataWrongFormatConfig.GenWithStackByArgs("must specify FIELDS [OPTIONALLY] ENCLOSED BY when use NULL DEFINED BY OPTIONALLY ENCLOSED")
```

Comment:
Maybe we should change to import into error

Reply:
this check is for LOAD DATA, IMPORT INTO is in `if` part